app/javascript/dashboard/components/widgets/conversation/components/SLACardLabel.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div
+    v-if="showSLACard"
+    class="flex items-center px-2 truncate border border-solid min-w-fit w-fit border-slate-75 dark:border-slate-700"
```

Comment:
```suggestion
    class="flex items-center px-2 truncate border w-fit border-slate-75 dark:border-slate-700"
```

- border by default is solid
- `min-w-fit` has the same effect as `w-fit`, we would need only one of those